### PR TITLE
Use `dropAllUsersFromDatabase` in tests

### LINKED ISSUE #1492: Implement `dropAllUsersFromDatabase` command
### What should be done?

With `usersInfo` implemented, we should implement the basic [`dropAllUsersFromDatabase ` command](https://www.mongodb.com/docs/manual/reference/command/dropAllUsersFromDatabase/).

Tests should use `createUser` and `usersInfo` commands, not `insert` and `delete`.

### Where?

https://github.com/FerretDB/FerretDB/tree/main/internal/handler

### Definition of Done

- handler updated;
- integration/compatibility tests added/updated;
- spot refactorings done.

### PR DESCRIPTION
**Pull request by henvic**
# Description

Closes #1492.

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [ ] I added/updated integration/compatibility tests (and they pass).
- [ ] I added/updated comments and checked rendering.
- [x] I made spot refactorings.
- [ ] I updated user documentation.
- [x] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

**Review by rumyantseva — DISMISSED** [review]
:shipit:

**Comment by AlekSi** [comment]
The next time just reopen the issue :)

File: integration/users/drop_user_test.go [hunk 1]
```diff
@@ -36,16 +36,9 @@ func TestDropUser(t *testing.T) {
...
-	if setup.IsMongoDB(t) {
-		require.NoError(t, collection.Database().RunCommand(ctx, bson.D{
-			{"dropAllUsersFromDatabase", 1},
-		}).Err())
-	} else {
-		// Erase any previously saved user in the database.
-		_, err := users.DeleteMany(ctx, bson.D{{"db", db.Name()}})
-		require.NoError(t, err)
-	}
+	require.NoError(t, collection.Database().RunCommand(ctx, bson.D{
```
Comment (AlekSi) [inline integration/users/drop_user_test.go:39]: ok, so `require` there…

File: integration/users/create_user_test.go [hunk 2]
```diff
@@ -38,16 +38,9 @@ func TestCreateUser(t *testing.T) {
...
-	if setup.IsMongoDB(t) {
-		assert.NoError(t, collection.Database().RunCommand(ctx, bson.D{
-			{"dropAllUsersFromDatabase", 1},
-		}).Err())
-	} else {
-		// Erase any previously saved user in the database.
-		_, err := users.DeleteMany(ctx, bson.D{{"db", db.Name()}})
-		require.NoError(t, err)
-	}
+	assert.NoError(t, collection.Database().RunCommand(ctx, bson.D{
```
Comment (AlekSi) [inline integration/users/create_user_test.go:41]: but `assert` there?..

Comment (henvic) [reply integration/users/create_user_test.go]: Changed!

**Review by AlekSi — APPROVED** [review]

**Pull request merged**